root/Makefile

```diff
@@ -230,6 +232,22 @@ docs:: pre-docs
 docs-publish:: pre-docs-publish
 	@./.bootstrap/shell/docs-publish.sh
 
+## stencil:         run stencil
+.PHONY: stencil
+stencil:: pre-stencil run-stencil post-stencil
+
+.PHONY: run-stencil
+run-stencil:
+	stencil --skip-update
+
+## restencil:       run stencil without updating module versions
+.PHONY: restencil
```

Comment:
Is this `restencil` only applied to the devbase module?

Reply:
Since this is in `root/Makefile` and not `Makefile`, this is applied to all stenciled repos using this version.

Reply:
I see. I take this is as the special case without updating devbase module, but it is no harm to add the feature for all stenciled repos.

Reply:
`make restencil` is the equivalent of someone running `stencil --frozen-lockfile`, for example if they change the `reportingTeam` in `service.yaml` and don't want to upgrade any stencil modules at the same time.